pages/common/cpio.md

```diff
@@ -1,16 +1,15 @@
 # cpio
 
-> Archiving utility.
+> Copies files to and from archives.
 
-- Create an archive from files
+- Take a list of file names from standard input and creates an archive from them:
```

Comment:
Actually, I would venture saying that "create" is only accurate if the archive does not yet exist (in which case this tool is kind of redundant to other archiving tools). We should probably lead out with an example of copying files into an existing archive.